src/module.cpp

```diff
@@ -628,6 +637,17 @@ instruction_ref module::find_dangling_reference() const
     return end();
 }
 
+bool is_div_zero(instruction ins) { return ins.name() == "@divzero"; }
+
+instruction_ref module::flag_division_by_zero() const
+{
+    auto divzero = std::count_if(begin(), end(), [](auto x) { return is_div_zero(x); });
```

Comment:
This should be using `find_if`

Reply:
don't we want to just see if one exists and halt compilation?